README.md

```diff
@@ -26,6 +26,7 @@ You can find the description for the available service types and their configura
 
 ### 3.8.1
 + Updated `saifish-core` version from `3.3.54` to `3.3.93`
++ New `fix` client property `encryptMethod`: password encryption support
```

Comment:
```suggestion
```
README in root should contain description for common changes for all sub-projects